projects/core/src/site-context/facade/currency-state-persistence.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core';
+import { StatePersistenceService } from '../../state/services/state-persistence.service';
+import {
+  getContextParameterDefault,
+  getContextParameterValues,
+} from '../config/context-config-utils';
+import { SiteContextConfig } from '../config/site-context-config';
+import { CURRENCY_CONTEXT_ID } from '../providers/context-ids';
+import { CurrencyService } from './currency.service';
+
+@Injectable({ providedIn: 'root' })
+export class CurrencyStatePersistenceService {
```

Comment:
PeristenceServices are semantically not facades. Let's move them from /facade to /services folder

Reply:
True